app/models/host/managed.rb

```diff
@@ -65,17 +65,17 @@ class Jail < ::Safemode::Jail
 
   attr_reader :cached_host_params
 
-  default_scope lambda {
-    org = Organization.current
-    loc = Location.current
-    conditions = {}
-    conditions[:organization_id] = org.subtree_ids if org
-    conditions[:location_id]     = loc.subtree_ids if loc
-    where(conditions)
-  }
-
-  scope :recent,      lambda { |*args| {:conditions => ["last_report > ?", (args.first || (Setting[:puppet_interval] + 5).minutes.ago)]} }
-  scope :out_of_sync, lambda { |*args| {:conditions => ["last_report < ? and enabled != ?", (args.first || (Setting[:puppet_interval] + 5).minutes.ago), false]} }
+  # default_scope lambda {
```

Comment:
that's a great question - I don't know why this breaks everything. it just
does. that's why I commented it out instead of deleting...